frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/EnvironmentAccordionBody.tsx

```diff
@@ -36,13 +36,13 @@ const StyledAccordionBodyInnerContainer = styled('div')(({ theme }) => ({
     },
 }));
 
-const StyledContentList = styled('ol')({
+export const StyledContentList = styled('ol')({
     listStyle: 'none',
     padding: 0,
     margin: 0,
 });
 
-const StyledListItem = styled('li', {
+export const StyledListItem = styled('li', {
```

Comment:
I'll consider extracting these into separate StrategyList components later. Depends on further refactoring. Making a task for it.